create.go

```diff
@@ -91,6 +92,16 @@ func create(containerID, bundlePath, console, pidFilePath string) error {
 		return err
 	}
 
+	// config.json provides a cgroups path that has to be used to create "tasks"
+	// and "cgroups.procs" files. Those files have to be filled with a PID, which
+	// is shim's in our case. This is mandatory to make sure there is no one
+	// else (like Docker) trying to create those files on our behalf. We want to
+	// know those files location so that we can remove them when delete is called.
+	cgroupsPath := "foo"
```

Comment:
Sorry I should have said that the PR is not ready, and that we need to find the way to retrieve cgroupsPath from the config.